src/test/java/com/github/splendor_mobile_game/websocket/config/EnvConfigTests.java

```diff
@@ -0,0 +1,119 @@
+package com.github.splendor_mobile_game.websocket.config;
+
+import com.github.splendor_mobile_game.websocket.config.exceptions.EnvFileNotFoundException;
+import com.github.splendor_mobile_game.websocket.config.exceptions.EnvRequiredValueNotFoundException;
+import com.github.splendor_mobile_game.websocket.config.exceptions.EnvValueWrongTypeException;
+import com.github.splendor_mobile_game.websocket.config.exceptions.InvalidConfigException;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class EnvConfigTests {
+    private final String testEnvConfigsDirectoryPath = "./testEnvConfigs/";
+
+    @Test
+    public void createDefaultEnvConfigTest(){
+
+        // 1. Setup data for the test
+        int port = 8887;
+        int connectionLostTimeoutSec = 120;
+        int pingIntervalMs = 1000;
+        int connectionCheckIntervalMs = 2000;
+        String logsDir = "./logs/";
+
+        // 2. Call the function you are testing.
+        try {
+            EnvConfig config = new EnvConfig();
```

Comment:
Unfortunately you should not check for the default config which is in the root directory, simply because it will change a lot.